src/Compilers/VisualBasic/Portable/VisualBasicCompilationOptions.vb

```diff
@@ -204,7 +204,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             _optionCompareText = optionCompareText
             _embedVbCoreRuntime = embedVbCoreRuntime
             _suppressEmbeddedDeclarations = suppressEmbeddedDeclarations
-            _parseOptions = parseOptions
+            _parseOptions = If(parseOptions, VisualBasicParseOptions.Default) ' Required to correctly check for the presence of feature flags.
```

Comment:
The `CheckFeatures` in the `Binder_Invocation`, if it is null lots of unrelated (as far as I can tell) test failures happen.